mobile/library/cc/engine_builder.cc

```diff
@@ -232,8 +232,8 @@ EngineBuilder& EngineBuilder::setNodeLocality(const NodeLocality& node_locality)
   return *this;
 }
 
-EngineBuilder& EngineBuilder::setAggregatedDiscoveryService(const std::string& address,
-                                                            const int port, std::string jwt_token,
+EngineBuilder& EngineBuilder::setAggregatedDiscoveryService(std::string address, const int port,
```

Comment:
since `address` is no longer a const ref, you'll want to do std::move when assigning it to `ads_addres_`